webServer.js

```diff
@@ -248,10 +249,14 @@ function start() {
       }
       if (assets.gennedFileNames === undefined) {
         var fileArray = [];
+        var regexpr = "{*min.js,*min.css,"+library.filename+"}";
```

Comment:
Can we improve the readability with better coding style here?
Some thing like: (just an example)
`var regexpr = "{*min.js,*min.css,"+library.filename+"}";`
⬇️ 
`var regexpr = "{*.min.js,*.min.css," + library.filename + "}";`